app/presenters/ht_registration_presenter.rb

```diff
@@ -52,13 +63,33 @@ def show_dsp_name
     action == :index ? link_to(dsp_name, ht_registration_path(id)) : dsp_name
   end
 
-  def show_env
-    return unless self[:env].present?
+  def show_detail_display_name
+    self[:env]["HTTP_X_SHIB_DISPLAYNAME"]
+  end
+
+  def show_detail_edu_person_principal_name
+    self[:env]["HTTP_X_SHIB_EDUPERSONPRINCIPALNAME"]\
```

Comment:
Trailing '\'?

Reply:
Will be fixing that with a force-push momentarily.